### DISCUSSION TITLE
How to ignore the invisible message

### DISCUSSION THREAD

**Discussion by SOVLOOKUP** [community/community#126807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

This subscription has always existed and cannot be closed, which really bothers me...
![image](https://github.com/community/community/assets/53158137/5009e791-2d40-431d-a252-8a1300a8210c)

![屏幕截图 2024-06-01 154823](https://github.com/community/community/assets/53158137/9c7c2f2c-ce6a-42ac-93b8-adfb30b95c7f)

**Comment by ebndev** [comment]
Hi @SOVLOOKUP, thanks for flagging this!

I am keeping a log of this issue as other users have mentioned this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission. You can keep an eye on our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

In the meantime, please see [this previous discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions.

Feel free to ask questions on the original threads if you run into an issue.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
